data/human/hails.txt

```diff
@@ -1670,6 +1670,47 @@ phrase "friendly civilian"
 		"The next event will be great!"
 		"That's real entertainment!"
 
+phrase "friendly civilian"
+	word
+		"I"
+		"Me and my friend"
+	word
+		" "
+	word
+		"had a job"
+		"used to spend time on"
+		"worked for years on"
+		"worked in the business of"
+	word
+		" "
+	word
+		"creating pixel art"
+		"creating computer games"
+		"creating 3D computer models"
+		"creating websites on the worlds wide web"
```

Comment:
```suggestion
		"creating websites on the World Wide Web"
```
While "web" or "internet" may not be capitalized today, I think "World Wide Web" still qualifies for capitalization.